bugsnag/notification.py

```diff
@@ -37,17 +37,12 @@ def request():
             status = resp.getcode()
 
             if status != 200:
-                bugsnag.log("Notification to %s failed, status %d" % (url,
-                                                                      status))
+                bugsnag.logger.warning(
```

Comment:
I've moved this to a warning message instead of info. This feels more of a warning that the notification did not successfully sent and not an informative message that it wasn't sent.